api/android/api/src/androidTest/java/org/nnsuite/nnstreamer/APITestCommon.java

```diff
@@ -0,0 +1,40 @@
+package org.nnsuite.nnstreamer;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
```

Comment:
@jaeyun-jung 
I have one question. Let's talk about the Doxygen documentation of the Java source code. 
Do we have to generate the Doxygen document in case of the Java code? or not? 

* Example:
![image](https://user-images.githubusercontent.com/82404/65284031-6d1f4a80-db73-11e9-94bf-7dfeaf395a4d.png)

Reply:
related issue #1562

Reply:
Acked-by: Geunsik Lim